samples/templates/default-azuredeploy.json

```diff
@@ -264,6 +274,58 @@
                 ]
             }
         },
+        {
+            "condition": "[equals(parameters('solutionType'), 'FhirServerSqlServer')]",
+            "name": "[variables('serviceName')]",
+            "type": "Microsoft.Sql/servers",
+            "apiVersion": "2015-05-01-preview",
+            "location": "[resourceGroup().location]",
+            "tags": {
+                "FhirServerSolution": "[parameters('solutionType')]"
+            },
+            "properties": {
+                "administratorLogin": "[uniqueString(resourceGroup().id)]",
```

Comment:
Is this setting the administrator user name? If it's generating a hash based on the resource group name, how would we know what that is since it's not being displayed or returned anywhere?

Reply:
I guess it's in the KeyVault so we could grab it from there.

Reply:
Depends on where you want to use it. It's stored in the connection string we put in keyvault and it's also displayed in the portal when you go to `Query Editor (Preview)`.

I'm not opposed to making it something known, but just thought this added one extra dimension that you had to know to get access.